src/__mocks__/pn.ts

```diff
@@ -52,12 +52,14 @@ export const aPNThirdPartyNotification = {
       content_type: "application/pdf",
       name: "Atto",
       url: `/delivery/notifications/received/${aPnNotificationId}/attachments/documents/0`,
+      category: "DOCUMENT",
     },
     {
       id: "JGQG-YPJT-AUQZ-202301-R-1_DOC1",
       content_type: "application/pdf",
       name: "Lettera di accompagnamento",
       url: `/delivery/notifications/received/${aPnNotificationId}/attachments/documents/1`,
+      category: "DOCUMENT",
```

Comment:
why not adding a test to check if providing no category results in a DEFAULT?
also, why not adding a test that shows that they can send an arbitrary category?